ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/defendant/DefendantMapper.java

```diff
@@ -51,6 +51,7 @@ public DefendantMapper(
         requireNonNull(claim, "claim must not be null");
 
         CCDRespondent.CCDRespondentBuilder respondentBuilder = CCDRespondent.builder();
+        respondentBuilder.servedDate(claim.getServiceDate());
```

Comment:
Shouldn't there be a corresponding change in `.from()`?

Reply:
yes, please add the relevant mapping in from as well.

Reply:
@andrewwa-kainos \ @sabahirfan the serviced date in the claim-store claim is not stored but in inferred from the issued on (+ 5 days) date, so it will be indirectly mapped upstream by the CaseMapper (which maps the issuedOn date). That's why I haven't introduced it in the from.

I can add `serviceDate` as an explicit field of Claim (and map it in the `from()`) but then we kind risk that issuedOn and serviceDate might go out of sync if somebody changes issuedOn after serviceDate is calculated. What do you think?

Reply:
Decided to go for the second approach, so that we will use the actual field mapped in CCD when mapping the claim back from CCD 👍